packages/components/src/lozenge/src/Lozenge.tsx

```diff
@@ -68,7 +72,8 @@ export function InnerLozenge({
         },
         text: {
             className: "o-ui-lozenge-text",
-            size: textSize(sizeValue)
+            size: textSize(sizeValue),
+            textTransform: textTransform(sizeValue)
```

Comment:
Pretty sure we decided when we implemented style props that Orbit component wouldn't use them to prevent rendering components with 47567 inline props.

Here you could easily do it in CSS, you already have the `o-ui-lozenge-text` class and also a size classe rendered.

Reply:
Totally, wanted to do the "clean" way as in CSS this could easily be overridden but I can do it in CSS no problem.

Reply:
Changed it to CSS